### PR TITLE
feat: walletconnect gas price input

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
## Description

taking over from https://github.com/shapeshift/web/pull/2718

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

related to https://app.dework.xyz/profile/southbear/board?taskId=f7ea0791-f07d-4af0-a694-a7d6e597e6e4

## Risk

* new functionality, will be tested entirely separately

## Testing

* this doesn't require testing and doesn't touch any existing code that isn't behind the new flag

## Screenshots (if applicable)

see original PR

### CODE REVIEW

File: src/assets/translations/en/main.json [hunk 1]
```diff
@@ -1438,6 +1438,22 @@
...
           "reject": "Reject"
+        },
+        "signTransaction": {
+          "advancedParameters": {
+            "title": "Advanced Parameters",
+            "alert": "For advanced use only. Modifying these settings could result in stuck transactions.",
+            "nonce": {
+              "title": "Nonce",
+              "placeholder": "Nonce...",
+              "tooltip": "explain nonce here"
```
Comment (0xdef1cafe) [inline src/assets/translations/en/main.json:1449]: @reallybeard @DiggyDiggy2

File: src/assets/translations/en/main.json [hunk 2]
```diff
@@ -1438,6 +1438,22 @@
...
+            },
+            "gasLimit": {
+              "title": "Gas Limit",
+              "placeholder": "Gas Limit...",
+              "tooltip": "explain gas limit here"
```
Comment (0xdef1cafe) [inline src/assets/translations/en/main.json:1454]: @reallybeard @DiggyDiggy2

File: src/components/DeFi/components/GasInput.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+import { FeeDataKey } from '@shapeshiftoss/chain-adapters'
+import type { FC } from 'react'
+import { Fragment, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { HelperTooltip } from 'components/HelperTooltip/HelperTooltip'
+import { getFeeTranslation } from 'components/Modals/Send/TxFeeRadioGroup'
+import { RawText, Text } from 'components/Text'
+
+type GasInputProps = {
+  value: FeeDataKey | undefined
```
Comment (0xdef1cafe) [inline src/components/DeFi/components/GasInput.tsx:24]: wrong

File: src/components/DeFi/components/GasInput.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  onChange(type: FeeDataKey): void
```
Comment (0xdef1cafe) [inline src/components/DeFi/components/GasInput.tsx:25]: needs to support custom

File: src/components/DeFi/components/GasInput.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+  const borderColor = useColorModeValue('gray.100', 'gray.750')
+  const bgColor = useColorModeValue('white', 'gray.850')
+  const translate = useTranslate()
+
+  const options = useMemo(
+    (): GasOption[] => [
+      {
+        value: FeeDataKey.Slow,
+        label: translate(getFeeTranslation(FeeDataKey.Slow)),
+        duration: '~ 10 mins',
```
Comment (0xdef1cafe) [inline src/components/DeFi/components/GasInput.tsx:46]: translations

File: src/components/DeFi/components/GasInput.tsx [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+                    </HStack>
+                  </Radio>
+                  <RawText color={option.color}>{option.amount}</RawText>
+                </HStack>
+                <Divider />
+              </Fragment>
+            ))}
+            <Box px={4} py={2} width='full'>
+              <HStack width='full'>
+                <Radio color='blue'>
```
Comment (0xdef1cafe) [inline src/components/DeFi/components/GasInput.tsx:121]: a non radio radio button

File: src/plugins/walletConnectToDapps/components/modal/ModalSection.tsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+        _hover={{ textDecoration: 'none' }}
+        textAlign='left'
+        justifyContent='flex-start'
+        leftIcon={!!icon ? <Box color='gray.500'>{icon}</Box> : undefined}
+        rightIcon={
+          isOpen ? <ChevronUpIcon color='gray.500' /> : <ChevronDownIcon color='gray.500' />
+        }
+        mb={4}
+        fontWeight='medium'
+        children={
```
Comment (gomesalexandre) [inline src/plugins/walletConnectToDapps/components/modal/ModalSection.tsx:39]: Prefer the JSX children as child node vs. the children prop but potato potato

**Review by gomesalexandre — APPROVED** [review]
LGTM,does what it says on the box as a dumb PR and self review comments have been tackled

**Pull request merged**